hloc/pairs_from_retrieval.py

```diff
@@ -67,6 +67,35 @@ def pairs_from_score_matrix(scores: torch.Tensor,
     return pairs
 
 
+def save_global_candidates_for_query(
```

Comment:
Why is this function needed? why isn't `pairs_from_retrieval.main` sufficient?

Reply:
You're right, `pairs_from_retreival.main` is sufficient. I will rewrite my example and push once I get the chance.